tasks/fix-cat2.yml

```diff
@@ -485,7 +485,9 @@
       - pamd
 
 - name: "MEDIUM | RHEL-07-010340 | PATCH | Users must provide a password for privilege escalation."
-  command: "true"
+  debug:
+    msg: "NOPASSWD entry for sudo: {{ item }}"
```

Comment:
okay, but better would be a remediation that actually fixes the issue.

Reply:
I'm good with a remediation, but I wasn't prepared to risk locking people out entirely by removing items. If replacing all instances of `NOPASSWD` with `PASSWD` is a guaranteed fix, I'm good with that, too.